libsolidity/interface/CompilerStack.h

```diff
@@ -455,6 +464,7 @@ class CompilerStack: boost::noncopyable
 	OptimiserSettings m_optimiserSettings;
 	RevertStrings m_revertStrings = RevertStrings::Default;
 	State m_stopAfter = State::CompilationSuccessful;
+	bool m_viaIR = false;
```

Comment:
Do you need to reset it to false in the reset function?

Reply:
Good point.